FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Tool/Program.cs

```diff
@@ -23,6 +25,8 @@ public class Program
     {
         public static async Task Main(string[] args)
         {
+            var assembly = Assembly.GetExecutingAssembly();
```

Comment:
We can remove these two lines as log has already been written in JobManager

Reply:
Updated.